site/src/pages/docs/server-grpc.mdx

```diff
@@ -271,6 +271,6 @@ For more information, see the official [gRPC Server Reflection tutorial](https:/
 
 [gRPC]: https://grpc.io/
 [gRPC-Web]: https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md
-[gRPC-Web-Client]: https://github.com/improbable-eng/grpc-web
+[gRPC-Web-Client]: https://github.com/grpc/grpc-node/tree/master/packages/grpc-js
```

Comment:
or link to the documentation site? https://grpc.io/docs/languages/web/quickstart/

Reply:
That's a good suggestion. 😉